README.md

```diff
@@ -13,19 +13,21 @@
 <!-- usage note: edit this section. -->
 
 Scientific news coverage shapes the public’s view of the current state of scientific findings and legitimizes experts.
-Through researching a story, journalists identify and interview a limited number of sources.
-These sources may come from a journalist’s research or through recommendations by other scientists.
-In either case, unconscious biases may influence who is identified as an expert to interview, possibly skewing the selection of interviewees.
-We analyzed more than 22,000 news articles published by _Nature_ to quantify possible disparities.
-Our analysis considered three possible sources of bias: gender, name origin, and country affiliation.
-To explore these sources of bias, we extracted cited authors’ names and affiliations, as well as extracted names of quoted speakers.
-We then used the names to predict gender and name origin of the authors and speakers.
-In our analysis, we found a bias towards male quotation, but quotation is trending toward equal representation at a faster rate than academic publishing.
-Interestingly, we found that the gender disparity in quotes was column-dependent, with the "career-features" column reaching gender parity.
+News stories frequently cite and quote a limited number of sources.
+These sources may be identified by the journalist’s research or by recommendations by other scientists.
+In both cases, biases may influence who is identified and ultimately included as an expert.
+We analyzed 22,001 news articles published by _Nature_ to quantify possible disparities.
+Our analysis considered three possible sources of disparity: gender, name origin, and country affiliation.
+To explore these sources of disparity, we extracted cited authors’ names and affiliations, as well as extracted names of quoted speakers.
+We then used the names to predict gender and name origin of the authors and speakers. <!-- maybe add caveat here -->
+In order to appropriately quantify the level of difference, we must identify an appropriate reference set for comparison.
+We chose first and last authors within primary research articles in _Nature_ and a subset of _Springer_ articles in the same time period as our comparator.
+In our analysis, we found a skew towards male quotation in _Nature_ news-related articles, but quotation is trending toward equal representation at a faster rate than first and last authorship in academic publishing.
+Interestingly, we found that the gender disparity in quotes was column-dependent, with the "Career-Features" column reaching gender parity.
```

Comment:
```suggestion
Interestingly, we found that the gender disparity in quotes was column-dependent, with the "Career Features" column reaching gender parity.
```
Should these have hyphens? I have no idea but asking just in case.

Reply:
I think you are right, there should be no hyphens. I originally had them because that's literally the string I query for, but in the article types listed on the website they do not have hyphens.

Reply:
replaced "-Feature" and "-Column" to " Feature" and " Column" in https://github.com/greenelab/nature_news_manuscript/pull/5/commits/7d3daa31e93537513db4f08ea4d7c13db39d2250